pulp_container/app/redirects.py

```diff
@@ -34,10 +35,17 @@ class FileStorageRedirects(CommonRedirects):
     A class which contains methods used for redirecting to the default django's file storage.
     """
 
-    def issue_tag_redirect(self, tag):
+    def issue_tag_redirect(self, tag, pk):
         """
         Issue a redirect for the passed tag.
         """
+        manifest_media_type = tag.tagged_manifest.media_type
+        if (
+            manifest_media_type not in get_accepted_media_types(self.request.headers)
+            and manifest_media_type != MEDIA_TYPE.MANIFEST_V1
+        ):
+            raise ManifestNotFound(reference=pk)
```

Comment:
Is it possible to use `tag.name` instead? If so, there is no need to have a an additional argument.